lms/static/sass/shared/_footer-edx.scss

```diff
@@ -44,6 +130,24 @@ footer#footer-edx-v3 {
     flex-wrap: wrap;
   }
 
+  * {
+    box-sizing: border-box;
+  }
+
+  .row {
+    display: flex;
+    flex-wrap: wrap;
+    margin-right: -15px;
+    margin-left: -15px;
```

Comment:
Negative margins should be avoided as much as possible, why the need for them here?